### DISCUSSION TITLE
Assumption of linearity

### DISCUSSION THREAD

**Discussion by juandavidgutier** [DoubleML/doubleml-for-py#176] [General] [answered]
Hi developers team,

I have a question about the models IRM and IIVM do they both impose the assumption that the outcome is a linear function of the treatment, or they are fully flexible non-parametric estimators of the relationship between the treatment and the outcome? And by the name, I assume that the models PLR and PLIV impose this linear assumption.

Thanks a lot for your answer.

**Comment by PhilippBach** [comment, marked as answer]
Hi @juandavidgutier , 

thanks for your questions, just to keep an overview on this discussion, I link your discussion opened in the R repository [here](https://github.com/DoubleML/doubleml-for-r/discussions/176)

We have an overview on all models in our [user guide](https://docs.doubleml.org/stable/guide/models.html)

You are completely right: The PLR and PLIV impose the assumption of a linear and additive (homogenous) effect and allows for nonlinearities only in terms of the nuisance components $l(X)$ and $m(X)$ (and in case of the PLIV $r(X)$)

The IRM and IIVM do not impose such an assumption and can hence be considered as nonparametric models.  For example, the allow the treatment effects to differ across individuals (aka heterogeneous treatment effects). Hence, if the assumptions of the PLR apply, you can expect the corresponding estimator to be more precise (smaller standard errors / tighter confidence intervals) than the IRM estimate. However, if these assumptions are not appropriate (like for example heterogeneity in the treatment effect for subgroups), the PLR will be biased whereas the IRM can be expected to be still consistent.

In case you look for more literature on the IRM you can look for the "doubly robust score" which is used for identification in the IRM.

I hope this answers your question 😄 

Best, 

Philipp

**Comment by juandavidgutier** [comment]
Hi [PhilippBach](https://github.com/PhilippBach)

Thanks a lot for your answer!!
